seatunnel-ci-tools/pom.xml

```diff
@@ -36,6 +37,18 @@
             <version>${javaparser.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
I find this function doesn't need import those libs. 

![image](https://github.com/user-attachments/assets/ec69dee7-7bd9-4925-85d2-aed593ee245c)

Could you explain why need import this libs?